src/App.js

```diff
@@ -1,35 +1,38 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Project from "./components/Project";
+import WorksList from "./components/WorksList";
+import data from "./data.json";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
+    let project;
+    data.forEach((proj) => {
+        return (project = proj);
+    });
```

Comment:
what are you trying to do here ? 
this will set the project variable to the last value of proj ...

Reply:
i end up not doing anything with it and i forgot to remove it